### ISSUE TITLE
both ios and android do not open the application

### ISSUE THREAD

**Issue by metinemredonmez** [inspireui/support#4870]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues:
- [] first issue. ios also does not open phone, I did pod instal but did not, how can I replace or replace phone with iphone 11 this problem disappears
- [] second issue second question, this app does not open Android phone with pixel 3
- [] third issue last question I did the Test flutter application from scratch there was this process, but it doesn't happen here

+ Product & version: the best package is woo commerce
+ Flutter (or React Native) version: latest version
+ Testing Device/Simulator: ios test phone 2nd generation, Android test phone pixel 3a

**_Step 2 (require): submit proof of purchasing the license on https://verify.inspireui.com **

<img width="1005" alt="Ekran Resmi 2020-09-04 10 14 20" src="https://user-images.githubusercontent.com/54211029/92211041-798dd780-ee98-11ea-9d45-eafc25d57384.png">
<img width="1136" alt="Ekran Resmi 2020-09-04 10 13 09" src="https://user-images.githubusercontent.com/54211029/92211050-7bf03180-ee98-11ea-8b6b-ac2441cc9b1f.png">

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by minhcasi** [comment]
The svg warning is not a bug, it is the solution to override flutter library on Flutter

Please try your best with the following solutions to fix the installing:
- Re-install the Flutter could help https://flutter.dev/docs/get-started/test-drive?tab=vscode
- Re-install the Android Studio SDK
- Clean up the Cache on Flutter https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1

**Comment by metinemredonmez** [comment]
I did what you said Android opened, ok, but ios still gives error running install error, 2 generation SE model how can I replace it with Iphone 11 I think it is connected to the error with the phone I opened, ror running pod install
Error launching application on iPhone SE (2nd generation). he's making his mistake, how can I fix it?
<img width="1136" alt="Ekran Resmi 2020-09-04 10 13 09" src="https://user-images.githubusercontent.com/54211029/92213970-890f1f80-ee9c-11ea-8c01-c03d8a279a4b.png">

**Comment by metinemredonmez** [comment]
sorry I threw the wrong screen picture. 
I did what you said Android opened, ok, but ios still gives error running install error, 2 generation SE model how can I replace it with Iphone 11 I think it is connected to the error with the phone I opened, ror running pod install
Error launching application on iPhone SE (2nd generation). he's making his mistake, how can I fix it?

<img width="1005" alt="Ekran Resmi 2020-09-04 10 14 20" src="https://user-images.githubusercontent.com/54211029/92214512-a9d77500-ee9c-11ea-9ca9-a228cbf2a051.png">

**Comment by minhcasi** [comment]
you could give a try on another simulator device, there is any issue your Pod library to download require library for the app.

**Comment by metinemredonmez** [comment]
I've tried it with another phone.

<img width="1073" alt="Ekran Resmi 2020-09-04 15 14 26" src="https://user-images.githubusercontent.com/54211029/92238219-63e0d800-eec1-11ea-8cd0-0a0728b48d51.png">

**Comment by metinemredonmez** [comment]
this ticket didn't close and I didn't get the answer to my question yet, please help change the last phone, but I still get the pod install error , I downloaded the app again and installed it from the beginning, but it doesn't change

**Comment by metinemredonmez** [comment]
Launching lib/main.dart on Emre iPhone’u in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: S9RPAM8224
Running pod install...
CocoaPods' output:
↳
  Preparing

Analyzing dependencies

Inspecting targets to integrate
  Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)
  Using `ARCHS` setting to build architectures of target `Pods-OneSignalNotificationServiceExtension`: (``)

Finding Podfile changes
  - Flutter
  - OneSignal
  - app_settings
  - apple_sign_in
  - connectivity
  - facebook_audience_network
  - firebase_admob
  - firebase_analytics
  - firebase_auth
  - firebase_core
  - firebase_database
  - firebase_messaging
  - flutter_facebook_login
  - flutter_statusbarcolor
  - flutter_webview_plugin
  - google_maps_flutter
  - google_sign_in
  - location
  - notification_permissions
  - onesignal_flutter
  - path_provider
  - rate_my_app
  - screen
  - share
  - shared_preferences
  - sqflite
  - uni_links
  - url_launcher
  - video_player
  - wakelock
  - webview_flutter

Fetching external sources
-> Fetching podspec for `Flutter` from `Flutter`
-> Fetching podspec for `app_settings` from `.symlinks/plugins/app_settings/ios`
-> Fetching podspec for `apple_sign_in` from `.symlinks/plugins/apple_sign_in/ios`
-> Fetching podspec for `connectivity` from `.symlinks/plugins/connectivity/ios`
-> Fetching podspec for `facebook_audience_network` from `.symlinks/plugins/facebook_audience_network/ios`
-> Fetching podspec for `firebase_admob` from `.symlinks/plugins/firebase_admob/ios`
-> Fetching podspec for `firebase_analytics` from `.symlinks/plugins/firebase_analytics/ios`
-> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
-> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
-> Fetching podspec for `firebase_database` from `.symlinks/plugins/firebase_database/ios`
-> Fetching podspec for `firebase_messaging` from `.symlinks/plugins/firebase_messaging/ios`
-> Fetching podspec for `flutter_facebook_login` from `.symlinks/plugins/flutter_facebook_login/ios`
-> Fetching podspec for `flutter_statusbarcolor` from `.symlinks/plugins/flutter_statusbarcolor/ios`
-> Fetching podspec for `flutter_webview_plugin` from `.symlinks/plugins/flutter_webview_plugin/ios`
-> Fetching podspec for `google_maps_flutter` from `.symlinks/plugins/google_maps_flutter/ios`
-> Fetching podspec for `google_sign_in` from `.symlinks/plugins/google_sign_in/ios`
-> Fetching podspec for `location` from `.symlinks/plugins/location/ios`
-> Fetching podspec for `notification_permissions` from `.symlinks/plugins/notification_permissions/ios`
-> Fetching podspec for `onesignal_flutter` from `.symlinks/plugins/onesignal_flutter/ios`
-> Fetching podspec for `path_provider` from `.symlinks/plugins/path_provider/ios`
-> Fetching podspec for `rate_my_app` from `.symlinks/plugins/rate_my_app/ios`
-> Fetching podspec for `screen` from `.symlinks/plugins/screen/ios`
-> Fetching podspec for `share` from `.symlinks/plugins/share/ios`
-> Fetching podspec for `shared_preferences` from `.symlinks/plugins/shared_preferences/ios`
-> Fetching podspec for `sqflite` from `.symlinks/plugins/sqflite/ios`
-> Fetching podspec for `uni_links` from `.symlinks/plugins/uni_links/ios`
-> Fetching podspec for `url_launcher` from `.symlinks/plugins/url_launcher/ios`
-> Fetching podspec for `video_player` from `.symlinks/plugins/video_player/ios`
-> Fetching podspec for `wakelock` from `.symlinks/plugins/wakelock/ios`
-> Fetching podspec for `webview_flutter` from `.symlinks/plugins/webview_flutter/ios`

Resolving dependencies of `Podfile`
  CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_c_1_2.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.15.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_c_7_9.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/7/9/Reachability/3.2/Reachability.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_0_3_5.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.31.1/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_9_b_5.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/7.1.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_b_3_c.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/7.1.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_d_4_0.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/d/4/0/GoogleSignIn/5.0.2/GoogleSignIn.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_a_d_d.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.10.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_2_1_5.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/1/5/FBAudienceNetwork/5.10.1/FBAudienceNetwork.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_f_4_e.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_5_9_a.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/5/9/a/Google-Mobile-Ads-SDK/7.65.0/Google-Mobile-Ads-SDK.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/5/9/a/Google-Mobile-Ads-SDK/7.63.0/Google-Mobile-Ads-SDK.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_e_3_b.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.8.0/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.7.0/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_0_8_4.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.2/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_6_1_e.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30906.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_6_3_6.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/3/6/FirebaseAuth/6.9.0/FirebaseAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/6/3/6/FirebaseAuth/6.7.1/FirebaseAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_8_b_d.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.10.1/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_c_e_3.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_e_2_1.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/2/1/FirebaseAnalytics/6.8.0/FirebaseAnalytics.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.14.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.14.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.9.2/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_8_3_c.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/8/3/c/FirebaseCoreDiagnostics/1.6.0/FirebaseCoreDiagnostics.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.9.2/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_0_b_5.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/b/5/PromisesObjC/1.2.10/PromisesObjC.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_2_d_6.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/d/6/FirebaseMessaging/4.6.2/FirebaseMessaging.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_d_8_2.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/d/8/2/FirebaseDatabase/6.5.0/FirebaseDatabase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/1/5/FBAudienceNetwork/5.8.0/FBAudienceNetwork.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/7/9/Reachability/3.2/Reachability.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/b/5/PromisesObjC/1.2.9/PromisesObjC.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/5.15.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/5.15.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/d/8/2/FirebaseDatabase/6.4.0/FirebaseDatabase.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_a_d_a.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/a/leveldb-library/1.22/leveldb-library.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/2/1/FirebaseAnalytics/6.7.0/FirebaseAnalytics.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_2_f_7.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.7.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.7.0/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.5.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/8/3/c/FirebaseCoreDiagnostics/1.5.0/FirebaseCoreDiagnostics.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_0_6_a.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/6/a/GoogleDataTransport/7.3.0/GoogleDataTransport.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/a/d/a/leveldb-library/1.22/leveldb-library.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/d/6/FirebaseMessaging/4.6.1/FirebaseMessaging.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_3_6_0.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/3/6/0/FirebaseInstanceID/4.6.0/FirebaseInstanceID.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_e_c_d.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/c/d/Protobuf/4.0.0-rc1/Protobuf.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/3/6/0/FirebaseInstanceID/4.5.1/FirebaseInstanceID.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.5.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/d/4/0/GoogleSignIn/5.0.2/GoogleSignIn.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_b_b_9.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: all_pods_versions_2_c_c.txt exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/c/c/GTMAppAuth/1.0.0/GTMAppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/0/6/a/GoogleDataTransport/7.1.0/GoogleDataTransport.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/e/c/d/Protobuf/3.12.0/Protobuf.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/2/c/c/GTMAppAuth/1.0.0/GTMAppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
  CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update

Comparing resolved specification to the sandbox manifest
  A AppAuth
  A FBAudienceNetwork
  A FBSDKCoreKit
  A FBSDKLoginKit
  A FMDB
  A Firebase
  A FirebaseAnalytics
  A FirebaseAuth
  A FirebaseCore
  A FirebaseCoreDiagnostics
  A FirebaseDatabase
  A FirebaseInstallations
  A FirebaseInstanceID
  A FirebaseMessaging
  A Flutter
  A GTMAppAuth
  A GTMSessionFetcher
  A Google-Mobile-Ads-SDK
  A GoogleAppMeasurement
  A GoogleDataTransport
  A GoogleMaps
  A GoogleSignIn
  A GoogleUtilities
  A OneSignal
  A PromisesObjC
  A Protobuf
  A Reachability
  A app_settings
  A apple_sign_in
  A connectivity
  A facebook_audience_network
  A firebase_admob
  A firebase_analytics
  A firebase_auth
  A firebase_core
  A firebase_database
  A firebase_messaging
  A flutter_facebook_login
  A flutter_statusbarcolor
  A flutter_webview_plugin
  A google_maps_flutter
  A google_sign_in
  A leveldb-library
  A location
  A nanopb
  A notification_permissions
  A onesignal_flutter
  A path_provider
  A rate_my_app
  A screen
  A share
  A shared_preferences
  A sqflite
  A uni_links
  A url_launcher
  A video_player
  A wakelock
  A webview_flutter

Downloading dependencies

-> Installing AppAuth (1.4.0)
  > Copying AppAuth from `/Users/metindonmez/Library/Caches/CocoaPods/Pods/Release/AppAuth/1.4.0-31bce` to `Pods/AppAuth`

-> Installing FBAudienceNetwork (5.8.0)
 > Http download
   $ /usr/bin/curl -f -L -o /var/folders/wg/mrj_w0_5003chs_3f3njr6f80000gn/T/d20200904-96602-1un4z9s/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

[!] Error installing FBAudienceNetwork
[!] /usr/bin/curl -f -L -o /var/folders/wg/mrj_w0_5003chs_3f3njr6f80000gn/T/d20200904-96602-1un4z9s/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Warning: Transient problem: HTTP error Will retry in 1 seconds. 2 retries 
Warning: left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Warning: Transient problem: HTTP error Will retry in 2 seconds. 1 retries 
Warning: left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 500 

/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:142:in `rescue in execute_command'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:139:in `execute_command'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:166:in `block in executable'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/http.rb:22:in `download_file'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/remote_file.rb:25:in `download!'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:82:in `block in download'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:157:in `block in ui_action'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:64:in `section'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:156:in `ui_action'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:80:in `download'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:108:in `download_source'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:75:in `download_request'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:171:in `download'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:154:in `block in uncached_pod'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:181:in `in_tmpdir'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:153:in `uncached_pod'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:33:in `download_pod'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:42:in `download'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer/pod_source_installer.rb:117:in `download_source'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer/pod_source_installer.rb:69:in `install!'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:557:in `install_source_of_pod'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:508:in `block (2 levels) in install_pod_sources'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:86:in `titled_section'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:507:in `block in install_pod_sources'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:490:in `each'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:490:in `install_pod_sources'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:249:in `block in download_dependencies'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:64:in `section'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:248:in `download_dependencies'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:157:in `install!'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/command/install.rb:52:in `run'
/Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'

Error output from CocoaPods:
↳
     % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                    Dload  Upload   Total   Spent    Left  Speed
     0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
   Warning: Transient problem: HTTP error Will retry in 1 seconds. 2 retries 
   Warning: left.
     0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
   Warning: Transient problem: HTTP error Will retry in 2 seconds. 1 retries 
   Warning: left.
     0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
   curl: (22) The requested URL returned error: 500 

[!] Automatically assigning platform `iOS` with version `10.3` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] Automatically assigning platform `iOS` with version `10.3` on target `OneSignalNotificationServiceExtension` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Exception: Error running pod install


and the debug part is like this??


The mDNS query for an attached iOS device failed. It may be necessary to disable the "Personal Hotspot" on the device, and to ensure that the "Disable unless needed" setting is unchecked under System Preferences > Network > iPhone USB. See https://github.com/flutter/flutter/issues/46698 for details.
Waiting for a connection from Flutter on iPhone 11...
Error connecting to the service protocol: connection to device ended too early


where is the problem ??

**Comment by minhcasi** [comment]
@metinemredonmez - could you try to run on Fresh Example Flutter to see if it have the same issue - https://flutter.dev/docs/get-started/test-drive ?

**Comment by metinemredonmez** [comment]
yes I've already tried it, it's happening I haven't had any trouble here just a nuisance on the ios side is present and still hasn't passed

**Comment by metinemredonmez** [comment]
** BUILD FAILED **


Xcode's output:
↳
ld: framework not found flutter_webview_plugin
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description

**Comment by minhcasi** [comment]
@metinemredonmez - pls double check again the solution on this link https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1 

Also, make sure using the latest Flutter stable channel `flutter channel stable` and latest Fluxstore `1.7.7`

**Comment by metinemredonmez** [comment]
I've already done it more than 10 times , I'm also using the latest version and the error still hasn't passed ! the error continues ! I think you should look at it why there is such a problem in the app ! I'm using Mac catalania, currently everything is up to date and the problem continues !

**Comment by minhcasi** [comment]
@metinemredonmez - could you give a try with the latest code on gitlab repo (pls submit here if you don't have account http://gitlab.inspireui.com/)

**Comment by metinemredonmez** [comment]
I sent the arrow, waiting for the purchase code and everything else.

**Comment by qozia3** [comment]
im having the same issue!

mac os Catalina. latest flutter latest Fluxstore woocommerce 1.7.7

**Comment by metinemredonmez** [comment]
Exactly yes, I have the same problem and we do what they say at the same time author , but this time my flutter does not open , I think I will delete it again and kuracam from the beginning , the same problem does not fix in any way , there are workarounds here, there is no solution ? I don't know how to do it

**Comment by metinemredonmez** [comment]
normally a very good application, bidet will be very good if I can run it, I think a very good application, but here is the problem this last error, vendor hopefully reads Our comments and saves us from this error :)

**Comment by qozia3** [comment]
`> Installing FBAudienceNetwork (5.8.0)
 > Http download
   $ /usr/bin/curl -f -L -o /var/folders/2z/tvy_w5297db6fc95m9ywy69c0000gp/T/d20200907-15544-hw8z1b/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

[!] Error installing FBAudienceNetwork
[!] /usr/bin/curl -f -L -o /var/folders/2z/tvy_w5297db6fc95m9ywy69c0000gp/T/d20200907-15544-hw8z1b/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
Warning: Transient problem: HTTP error Will retry in 1 seconds. 2 retries 
Warning: left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Warning: Transient problem: HTTP error Will retry in 2 seconds. 1 retries 
Warning: left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 500 
`


guys please FIX this. the url doesnt exist
see attached photo
<img width="976" alt="Screen Shot 2020-09-07 at 9 58 18 PM" src="https://user-images.githubusercontent.com/69852582/92412654-5e032500-f155-11ea-851d-daa70609fc2d.png">

**Comment by metinemredonmez** [comment]
I usually had the same error , I installed project from the beginning , for example, I said flutter clean, then look at this article about the flutter pod file pod, I installed them from the beginning, but this time other files gave an error ? can you try to rebuild it? or we'll wait for an update.


https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1

**Comment by qozia3** [comment]
@metinemredonmez I found the problem!

If you still has an issue with pod installing FBaudience then use a VPN.
It worked for me.

**Comment by metinemredonmez** [comment]
I'm really, really glad ı m happy, I opened ios u , but I have another computer that I need to install from the beginning can you help me with this , about connecting to my computer remotely , I can give facebook or whats up, I really appreciate it ? is very important to me

**Comment by qozia3** [comment]
> I'm really, really glad ı m happy, I opened ios u , but I have another computer that I need to install from the beginning can you help me with this , about connecting to my computer remotely , I can give facebook or whats up, I really appreciate it ? is very important to me

I think it's against the rules I'm sorry. 
Contact the developers at inspireui. 
They might help you :)

**Comment by minhcasi** [comment]
@metinemredonmez - you could try to remove the FBAudienceNetwork library to fix the build issue if you could download from your side

**Comment by metinemredonmez** [comment]
ok understand but , I sent it from GitLab, waiting for it, but I think it will open this time, this is what we're talking about, right ?

**Comment by metinemredonmez** [comment]
yes, the problem is solved, my fear is that I want to activate it with my own apple developer account this time both account and one signal give an error because I created a P 12 file for both firebase and one signal setrifika now I'm afraid to integrate them because the possibility of making an error. ? how can I do them, the app is opened

**Comment by metinemredonmez** [comment]
'/Users/metindonmez/Library/Developer/Xcode/DerivedData/Runner-ftknxnfylwrepodndrkjrdxsyicj/Build/Products/Debug-iphoneos/Pods_OneSignalNotificationServiceExtension.framework/Pods_OneSignalNotificationServiceExtension(Pods-OneSignalNotificationServiceExtension-dummy.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/metindonmez/Library/Developer/Xcode/DerivedData/Runner-ftknxnfylwrepodndrkjrdxsyicj/Build/Products/Debug-iphoneos/Pods_OneSignalNotificationServiceExtension.framework/Pods_OneSignalNotificationServiceExtension' for architecture arm64

**Comment by metinemredonmez** [comment]
I tried it on my own phone this way , my phone iPhone 6 , ,ios update problem , or do I need to throw the one signal file into it, or does the certificate I created in the apple developer account automatically communicate and import this certificate ?

**Comment by metinemredonmez** [comment]
Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.

		Embedded Binary Bundle Identifier:	com.metinemre.app.onesignal
		Parent App Bundle Identifier:		com.metinemre.mstore.boutiqueshop

**Comment by metinemredonmez** [comment]
Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.

		Embedded Binary Bundle Identifier:	com.metinemre.app.onesignal
		Parent App Bundle Identifier:		com.metinemre.mstore.boutiqueshop

**Comment by minhcasi** [comment]
@metinemredonmez - pls double check if you have replaced the bundle id correct and clean up the caching? https://docs.inspireui.com/fluxstore/delivery/#12-change-the-app-bundle-id-package-name

**Comment by qozia3** [comment]
> Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.
> 
> ```
> 	Embedded Binary Bundle Identifier:	com.metinemre.app.onesignal
> 	Parent App Bundle Identifier:		com.metinemre.mstore.boutiqueshop
> ```

The one signal bundle I'd should be :

com.metinemre.mstore.boutiqueshop.onesignal

It should be the same name as the app bundle Id+onesignal at the end.

If your app bundle id is:
Com.something.mstore.
Then onesignal bundle ID should be:
com.something.mstore.onesignal

**Comment by metinemredonmez** [comment]
ok, OK, I have solved this problem ,but I have a question like this , can I provide this process from my own hosting company , instead of digital ocean for website integration, word press provides site service in my own company after all, what is the difference between digital ocean ? thanx

**Comment by minhcasi** [comment]
Yes, you could use any hosting provider

**Comment by minhcasi** [comment]
@metinemredonmez - you could use any theme as the Paypal and Stripe base on core WooCommerce library (work separate), the theme is the only effective for the look and feels on your site - some theme support extra plugins that help on the frontend side.

**Comment by minhcasi** [comment]
Let us know if you have any question - thank you!

**Comment by metinemredonmez** [comment]
I can't answer it on my own phone Live What do you think is the reason

<img width="1054" alt="Ekran Resmi 2020-09-15 20 18 14" src="https://user-images.githubusercontent.com/54211029/93243331-b7400980-f790-11ea-8ebd-ece61f9372b7.png">

**Comment by metinemredonmez** [comment]
IPhone does not open on my own phone , android also opens , but the room receives an error in the payment part, I wonder if I have finished the integration of the website because of this olabilir? other than that, 3. error launch I want to change the screen the video also does not open when I change the current

![WhatsApp Image 2020-09-15 at 20 22 00](https://user-images.githubusercontent.com/54211029/93243799-6381f000-f791-11ea-8db1-28e365dd6819.jpeg)
